src/components/Form.js

```diff
@@ -320,7 +320,26 @@ function Form(props) {
                         }
                     },
                 });
-            }),
+            });
+
+            _.each(inputRefs.current, (inputRef, inputID) => {
```

Comment:
I think it makes sense to add a comment for what we're doing here.

Reply:
Okay, will add it.

Reply:
Done - https://github.com/Expensify/App/pull/21950/commits/38659cb9336ba19844cbba4c15d20ca6b070d54d